examples/win32-front-proxy/requirements.in

```diff
@@ -0,0 +1,3 @@
+Flask==2.0.3
```

Comment:
no need to hardcode the version here - it will get locked in the .txt file

you dont need the deps below either as our only actual dep is flask - these will still be added in .txt file

Reply:
actually it seems liike `requests` is specified so at least keep that - im wondering if it was only specified as a dep of flask or whether its needed - not sure aout the psycopg2-binary - it wasnt in the original - was it needed ?

Reply:
It is from auto generate.

/cc @envoyproxy/windows-dev